executor/aggfuncs/func_count_test.go

```diff
@@ -103,6 +103,20 @@ func (s *testSuite) TestCount(c *C) {
 
 func (s *testSuite) TestMemCount(c *C) {
 	tests := []aggMemTest{
+		buildAggMemTester(ast.AggFuncCount, mysql.TypeLonglong, 5,
+			aggfuncs.DefPartialResult4CountSize, distinctUpdateMemDeltaGens, true),
```

Comment:
`count funcs` are not distinct, please use `defaultUpdateMemDeltaGens` and set `isDistinct` false